src/app/layout/publish-carpool/publish-carpool.component.html

```diff
@@ -45,7 +45,7 @@ <h1 class="text-3xl" >Publier un trajet</h1>
 
         <mat-form-field subscriptSizing="dynamic" class="w-full">
             <mat-label>Date du départ</mat-label>
-            <input matInput required formControlName="departure_date" [matDatepicker]="picker" id="dateDeparture"  [matDatepickerFilter]="dateFilter">
+            <input matInput required formControlName="departure_date" [matDatepicker]="picker" id="dateDeparture_id"  [matDatepickerFilter]="dateFilter">
```

Comment:
```suggestion
            <input matInput required formControlName="departure_date" [matDatepicker]="picker" id="date_departure_id"  [matDatepickerFilter]="dateFilter">
```
Pour que ça soit homogène avec les autres id's